README.md

```diff
@@ -50,3 +72,19 @@ yarn deploy
 ```
 
 This command will deploy your Cloudflare Workers application and make it accessible.
+
+## Testing Telegram Bot
+
+1. Install Bot on your group
+2. Edit Bot in [@BotFather](https://t.me/botfather) dashboard (On Telegram) - Add slash commands `/start`
+3. Copy your Worker URL on Github and add `/registerWebhook` as suffix, ex: `https://WORKER_URL/registerWebhook` - it should return `Ok` if everything works then you can move to the next step
+4. Use the private chat as an admin to trigger the `/start` command and link a Github Repo to any of the listed channels
+
+![Screenshot 2023-08-18 at 8 15 28 PM](https://github.com/ubiquity/telegram-ubiquibot/assets/51956013/4e3d9313-af18-406e-9c91-3efcc372eb54)
+
+4. Members of your group can use `/register` command (sent to the group) and a link will be sent to their inbox to bind their telegram with their Github accounts
```

Comment:
I'm not sure if this is easy for you to integrate (perhaps in another bounty) but I recall that in the telegram bot settings (botfather) there is a way for the bot to display prompts in the group chat that is only visible to the invoker. 

I've also seen this in large group chats where the bot verified the new person joining is a human (it will prompt them to solve some type of puzzle or to click a link to solve a captcha) this is only visible to the person who just joined the group. 


This feature is very useful for everything this bot does:

1. Requesting to make a task (will not double notify everyone, from the task creator and the bot both sending messages)
2. The register command (so that they are not frontran)

Reply:
> This feature is very useful for everything this bot does:

I'd look into it, perhaps in another bounty.. the register command wouldn't be so easy to frontrun without access to the DB though.. it heavily relies on the session key